### DISCUSSION TITLE
Inputting an Array List for Testing in Azure AI Studio

### DISCUSSION THREAD

**Discussion by l0ginp** [microsoft/promptflow#3634] [Q&A] [answered]
How can I input an array list for testing purposes in the chatbox or another area within Azure AI Studio?

Example need to send this value for test
`["hello", "https://github.com", "https://github.com/test.png"]`

<img width="1249" alt="Screenshot 2567-08-08 at 17 49 24" src="https://github.com/user-attachments/assets/201c4f0d-30c4-44f2-b3b1-805b43904e31">

Now it needs to be deployed first before it can be tested via API.

**Comment by brynn-code** [comment, marked as answer]
Hello, thanks for reaching us. The github issue and discussion here only for tracking the open source promptflow problems. For the azure version, please provider feedback via: https://microsoft.github.io/promptflow/cloud/azureai/faq.html#how-to-create-feedback-for-azure-portal-experience, our PM will help triage it and contact you, thanks.

**Reply by l0ginp** [reply]
Thank you for your response. 
I've been trying to find a place to give feedback to Azure for a while.  😅
